frontend/src/app/document/document-preview/document-preview.component.ts

```diff
@@ -12,8 +12,15 @@ export class DocumentPreviewComponent {
     @Input() document: FoundDocument;
     @Input() page: DocumentPage;
 
+    actionIcons = actionIcons;
     documentIcons = documentIcons;
 
+    get documentUrl(): string[] {
```

Comment:
Technically missing `void` return type, but I assume a situation with `this.document` doesn't really arise